.github/workflows/backport.yml

```diff
@@ -0,0 +1,27 @@
+name: Backport
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    name: Backport
+    runs-on: ubuntu-latest
+    # Only react to merged PRs for security reasons.
+    # See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target.
+    if: >
+      github.event.pull_request.merged
+      && (
+        github.event.action == 'closed'
+        || (
+          github.event.action == 'labeled'
+          && contains(github.event.label.name, 'backport')
+        )
+      )
+    steps:
+      - uses: sourcegraph/backport@v2
+        with:
+          github_token: ${{ secrets.BACKPORT_GITHUB_TOKEN }}
+          label_pattern: '^cherrypick (?<base>jetbrains-v\d+\.\d+\.x)$'
```

Comment:
Is this the label convention we want?

Reply:
Oh shit, it should be `backport jetbrains-v<version>`

Fixed

Reply:
I spoke to @kalanchan and he said at the time, that for jetbrains the release branch should be `jetbrains-v<version>`, like how in `sourcegraph/cody` it's `vscode-v<version>`